RepresentsMe/ViewControllers/Officials/OfficialDetails/OfficialDetailsViewController.swift

```diff
@@ -49,9 +50,29 @@ class OfficialDetailsViewController: UIViewController {
     /// Starts a call with the official based on the phone number provided in
     /// the database
     @IBAction func callButtonPressed(_ sender: Any) {
-        if let phoneNumber = URL(string: "tel://\(official!.phones[0])") {
-            UIApplication.shared.open(phoneNumber)
-        }
+        let phones = official!.phones.map( { $0.filter("01234567890".contains) } )
```

Comment:
make this
if let phones = official?.phones.map( { $0.filter("01234567890".contains) } ) {
}
and put the rest of the method in this conditional, so the app wont crash if official is nil for some reason